projects/oak/build.sh

```diff
@@ -43,31 +43,34 @@ fi
 readonly NO_VPTR='--copt=-fno-sanitize=vptr --linkopt=-fno-sanitize=vptr'
 
 readonly FUZZER_TARGETS=()
+readonly ENABLED=false
```

Comment:
This is wrong way, please check example here, it needs to be in projecy.yaml.
https://github.com/google/oss-fuzz/blob/71f4914c453d1cba4f13dc331830378939ee84d9/projects/bad_example/project.yaml#L7

Reply:
We already have `disabled: True` in our yaml file (as of #4053), but it does not seem to work. [This](https://oss-fuzz-build-logs.storage.googleapis.com/log-175032d8-fa97-4155-a3aa-4160f40b5ff3.txt) is a failure report from Jul 3, 2020, after #4053 was merged. 

I still think the change in this PR is required, because currently running `build.sh` gives the following error:
```
/src/build.sh: line 47: FUZZER_TARGETS[@]: unbound variable
```

Reply:
Looks like jenkins builder was still building it, so i deleted it. i will merge this pr as well although it does not matter now. we would appreciate if some time later you can enable back oak fuzzing.

Reply:
Many thanks. We have an [internal issue](https://github.com/project-oak/oak/issues/893) to enable fuzz testing for our project, which is now mostly in Rust.